docker-compose.yml

```diff
@@ -21,4 +21,5 @@ services:
       - ./dbschema:/app/dbschema
     environment:
       - DATABASE_URL=postgresql+psycopg2://root:root@db/perfectfit
+      - TEST_USER_ID=41482
```

Comment:
Not sure why this is hard coded here.

Reply:
this is because you need this variable to be loaded into the environment. If the system is started from the main, we have the variable in the .env file, but if you just need to run just the virtual-coach-db container, than we have to specify it in the docker-compose file